src/DGtal/dec/DiscreteExteriorCalculus.h

```diff
@@ -199,6 +204,14 @@ namespace DGtal
      */
     DiscreteExteriorCalculus();
 
+    /**
+     * Init Khalimsky space boundaries.
+     * @tparam TDomain type of digital domain.
+     * @param domain domain used to initialize Khalimsky space.
+     */
+    template <typename TDomain>
+    void
+    initKSpace(const TDomain& domain);
```

Comment:
The domain must be "ConstAlias"  (some CDomain models store explicitly the point set --domain from set-- and we don't want to copy all the data)